app/views/application/_icon_circled_check.html

```diff
@@ -1,14 +1,6 @@
-<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 48.2 (47327) - http://www.bohemiancoding.com/sketch -->
-    <title>icn-check</title>
-    <desc>Created with Sketch.</desc>
-    <defs></defs>
-    <g id="legacy-oauth-patch-2" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(-625.000000, -228.000000)">
-        <g id="dialog" transform="translate(364.000000, 143.000000)">
-            <g id="icn-check" transform="translate(264.000000, 88.000000)">
-                <polygon id="Path-3" fill="#00BA99" points="13 35.2333261 17.106044 31.5025196 27.2748634 40.3526342 50.1751806 20 54 23.5034343 29.0434565 49 27.2748634 49"></polygon>
-                <circle id="Oval-4" stroke="#00BA99" stroke-width="5" cx="33" cy="33" r="33"></circle>
-            </g>
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="72" height="72" viewBox="0 0 72 72">
```

Comment:
@lgebhardt pinged me asking if this SVG file was in the right format. It was missing the correct formatting and I've fixed this in a separate commit of this PR.

@lgebhardt @jenn-rhim before exporting SVGs from Sketch you should install this compressor plugin: https://github.com/BohemianCoding/svgo-compressor not only does it save some bytes on the size, but additionally it removes the use of `id` attributes. When we have multiple SVGs on the page, the `id` attributes of uncompressed files can cause conflicts between the rendered images. Using this plugin is really a requirement of adding SVG to the site.